pkgs/development/tools/nufmt/default.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  ...
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "nufmt";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "nushell";
+    repo = "nufmt";
+    rev = "796ee834c1e31ead4c5479bf2827a4339c5d61d1";
+    hash = "sha256-BwKLl8eMCrqVt9PA5SHAXxu3ypP2ePcSuljKL+wSkvw=";
+  };
+
+  cargoSha256 = "sha256-JL1Ot2ldvieRM/SNLCC2YZ5b2hD98GXp+H4T/OFuKfk=";
```

Comment:
I got a different checksum here:

```
┏━ 2 Errors:
┃ error: builder for '/nix/store/v0jx1sgv5v0c6axjpcq87xmpgvalgspx-nufmt-0.1.0-vendor.tar.gz.drv' failed with exit code 1
┃ error: build of '/nix/store/v0jx1sgv5v0c6axjpcq87xmpgvalgspx-nufmt-0.1.0-vendor.tar.gz.drv' on 'ssh-ng://nix@ryan.dse.in.tum.de' failed: hash mismatch in fixed-output derivation '/nix/store/v0jx1sgv5v0c6axjpcq87xmpgvalgspx-nufmt-0.1.0-vendor.tar.gz.drv':
┃          specified: sha256-JL1Ot2ldvieRM/SNLCC2YZ5b2hD98GXp+H4T/OFuKfk=
┃             got:    sha256-SsOnuDnWkSRA5l8D04GqSwrAlijgWZNA8VdrZvB86SQ=
┣━ Dependency Graph:
┃       ┌─ ✔︎ source ⏱︎ 1s
┃    ┌─ ⚠︎ nufmt-0.1.0-vendor.tar.gz failed with exit code 1 after ⏱︎ 17s in fixupPhase
┃ ┌─ ⏳︎ nufmt-0.1.0
┃ ⏳︎ review-shell
┣━━━ Builds           │ Uploads   │ Host
┃        │ ✔︎ 1 │      │     │     │ localhost
┃        │     │      │     │ ↑︎ 1 │ ssh-ng://nix@ryan.dse.in.tum.de
┗━ ∑︎ ⏵︎ 0 │ ✔︎ 1 │ ⏳︎ 2 │ ↑︎ 0 │ ↑︎ 1 │ ⚠︎ Exited after 1 build failures at 20:45:15 after 25s

Link to currently reviewing PR:
https://github.com/NixOS/nixpkgs/pull/257297

1 package failed to build:
nufmt
```

Reply:
Oh, i changed to use pname without thinking it would affect the hash, silly me